src/govuk/helpers/_colour.scss

```
@@ -75,7 +75,8 @@
 /// Make a colour darker by mixing it with black
 ///
 /// @param {Colour} $colour - colour to shade
-/// @param {Number} $percentage - percentage of `$colour` in returned colour
+/// @param {Number} $percentage - alpha percentage applied to black
+/// on top of `$colour`
```

I'm not too hot on my colour theory but I'm not sure that it makes sense to talk about 'alpha' here as that tends to relate to transparency? Compared to the original version I'm not sure I really understand this what this means.

Does this need to be more complicated than just changing `$colour` to `black` or `white`?

```suggestion
/// @param {Number} $percentage - percentage of black in returned `$colour`
```

---

In my head the white/black was overlaying `$colour` which is why my head went to alpha, but that's not actually what's happening. The clues in the name that it's using `mix`.

Re: wording, I wonder if it's worth adding a smidge more detail. The way I read the original line, the "returned colour" is the colour that is output by these mixins so maybe specifying that the `$percentage` controls how much black/white is mixed with `$colour` to produced the returned colour.

I'm personally a fan of @querkmachine's suggestion above:

>percentage of black/white to mix with $colour

...or possibly an expanded version:

>percentage of black/white to mix with $colour in returned colour

---

Might be worth talking to our designers to see how they think about tints and shades and if there's a common vocabulary we should be using here.

---

I bothered @christopherthomasdesign about this and he's concurred that my first suggestion, aka: a minor re-wording of @querkmachine's suggestion, is the winner. I'll therefore update this PR in a tic to reflect this.